resources/org/javarosa/xpath/expr/randomize.xml

```diff
@@ -4,20 +4,24 @@
         <h:title>Randomize Test</h:title>
         <model>
             <instance>
-                <randomize id="randomize">
-                    <fruit1/>
-                    <fruit2/>
-                    <seededFruit1/>
-                    <seededFruit2/>
-                    <randomValue/>
-                    <seededRandomValue/>
+                <data id="randomize">
+                    <no-seed-random-value/>
```

Comment:
I would have used some groups here to make it even more readable/structured, but then it wouldn't work with the current testing tools (they assume everything is at the root instance level)